elasticsearch/run.sh

```diff
@@ -3,12 +3,21 @@
 mkdir -p /elasticsearch/$CLUSTER_NAME
 ln -s /etc/elasticsearch/keys/searchguard.key /elasticsearch/$CLUSTER_NAME/searchguard_node_key.key
 
-sed --in-place=.bak 's/searchguard.ssl.transport.http.enabled: true/searchguard.ssl.transport.http.enabled: false/' /usr/share/elasticsearch/config/elasticsearch.yml
-sed -i 's/allow_all_from_loopback: false/allow_all_from_loopback: true/' /usr/share/elasticsearch/config/elasticsearch.yml
-echo "" >> /usr/share/elasticsearch/config/elasticsearch.yml
-echo "network.host: 127.0.0.1" >> /usr/share/elasticsearch/config/elasticsearch.yml
+# do an initial run to initialize the SearchGuard ACL
+sed --in-place=.bak '
+	s/searchguard.ssl.transport.http.enabled: true/searchguard.ssl.transport.http.enabled: false/
```

Comment:
this is so that we can seed the ACLs without needing to use client cert/keys.  This will most likely look different/cleaner/less hacky post 3.1 when we seed in a DC hook instead